src/region.js

```diff
@@ -84,9 +84,11 @@ const Region = MarionetteObject.extend({
     }
   },
 
-  _attachView(view) {
+  _attachView(view, options = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!this.getOption('replaceElement');
+    const shouldReplaceEl = _.defaults(options, {
```

Comment:
Does kind of weird things if you pass in bad data.. I think I'd always expect `option.shouldReplaceEl`'s value to override if not undefined.